src/utils/tracking.js

```diff
@@ -2,7 +2,7 @@ import { init, push } from '@socialgouv/matomo-next';
 import { Auth } from 'aws-amplify';
 import Env from './environment';
 
-const MATOMO_URL = 'https://biomage.matomo.cloud/';
+const MATOMO_URL = 'https://biomage.matomo.cloud';
```

Comment:
The issue with heatmap not showing up is here. The problem is, the `socialguv/matomo-next` plugin that we're using adds an extra `/` for paths. See https://github.com/SocialGouv/matomo-next/blob/master/src/index.ts#L77